docs/machine-learning/tutorials/object-detection-onnx.md

```diff
@@ -0,0 +1,713 @@
+---
+title: 'Tutorial: Detect objects using deep learning with ONNX and ML.NET'
+description: This tutorial illustrates how to use a pre-trained ONNX deep learning model in ML.NET to detect objects in images.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 07/29/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to use ML.NET so that I can use a pre-trained model in an object detection scenario to detect objects in images using ONNX.
+---
+
+# Tutorial: Detect objects using ONNX in ML.NET
+
+Learn how to use a pre-trained ONNX model in ML.NET to detect objects. The pre-trained model was trained to find and classify objects in an image.
+
+Training an object detection model from scratch requires setting millions of parameters, a large amount of labeled training data and a vast amount of compute resources (hundreds of GPU hours). Using a pre-trained model allows you to shortcut the training process.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Understand the problem
+> * Learn what ONNX is and how it works with ML.NET
+> * Understand the model
+> * Reuse the pre-trained model
+> * Detect objects with a loaded model
+
+## Pre-requisites
+
+- [Visual Studio 2017 15.6 or later](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2017) with the ".NET Core cross-platform development" workload installed.
+- [Microsoft.ML NuGet Package](https://www.nuget.org/packages/Microsoft.ML/)
+- [Microsoft.ML.ImageAnalytics NuGet Package](https://www.nuget.org/packages/Microsoft.ML.ImageAnalytics/)
+- [Microsoft.ML.OnnxTransformer NuGet Package](https://www.nuget.org/packages/Microsoft.ML.OnnxTransformer/)
+- [Tiny YOLOv2 pre-trained model](https://github.com/onnx/models/tree/master/tiny_yolov2)
+- [Netron](https://github.com/lutzroeder/netron) (optional)
+
+## ONNX object detection sample overview
+
+This sample creates a .NET core console application that detects objects within an image using a pre-trained deep learning ONNX model. The code for this sample can be found on the [dotnet/machinelearning-samples repository](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx) on GitHub.
+
+## What is object detection?
+
+Object detection is a computer vision problem. While closely related to image classification, object detection performs image classification at a more granular scale. When an image is processed, there's first a check on whether there's an object present or not. If so, the detected object is classified and using object localization, the location of that object is returned along with the class. Typically, you want to use this approach on images where there are multiple objects of different types so that each is individually recognized. 
```

Comment:
Suggest simplifying to:
`Object detection both locates _and_ categorizes entities within images. Use object detection when images contain multiple objects of different types.`